ibllib/io/session_params.py

```diff
@@ -470,7 +470,7 @@ def prepare_experiment(session_path, acquisition_description=None, local=None, r
         return
     # Determine if user passed in arg for local/remote subject folder locations or pull in from
     # local param file or prompt user if missing
-    params = misc.create_basic_transfer_params(transfers_path=local, remote_data_path=remote)

```

Comment:
@k1o0 seems like a typo to me could you please have a look ?

Reply:
Good catch! Yes it does appear to be a typo.